### DISCUSSION TITLE
homebrew please

### DISCUSSION THREAD

**Discussion by virajp** [datreeio/datree#90] [Ideas]
It will be great if datree is available via homebrew

**Comment by eyarz** [comment]
When we launched this project we decided to go with a one-line install script (`curl https://get.datree.io | /bin/bash`), instead of homebrew, because it was easier to create :sweat_smile:

I agree that (**also**) having homebrew would be great, but I'm also wondering if there is a use case for someone that can only use homebrew, or it's just another "nice to have" option for installing Datree...

WDYT?

**Reply by virajp** [reply]
It's easier to have managed installation and updates. Also from security point of view, running shell scripts from internet is not really considered safe (though people do it all the time).

**Reply by eyarz** [reply]
I think you're right so we decided to prioritize that - https://github.com/datreeio/datree/issues/115

**Comment by eyarz** [comment]
officially supported :)
```
brew tap datreeio/datree
brew install datree
```
